code/modules/mob_spawn/mob_spawn.dm

```diff
@@ -172,9 +173,12 @@
 	user.log_message("became a [prompt_name].", LOG_GAME)
 	uses -= 1 // Remove a use before trying to spawn to prevent strangeness like the spawner trying to spawn more mobs than it should be able to
 
-	if(!(create(user)))
-		message_admins("[src] didn't return anything when creating a mob, this might be broken! The use of the spawner it would have taken has been refunded.")
-		uses += 1 // Oops! We messed up and somehow the mob didn't spawn, but that's alright we can refund the use.
+	var/created = create(user)
+	if(!created)
+		if(isnull(created)) // If we explicitly return FALSE instead of just not returning a mob, we don't want to spam the admins
+			message_admins("[src] didn't return anything when creating a mob, this might be broken! The use of the spawner it would have taken has been refunded. [ADMIN_FLW(src)]")
```

Comment:
Why is this a message_admins instead of something like a runtime or crash? (I forget exactly what we use)

Reply:
I guess plausibly an admin might want to know because a player is potentially going to mention the fact that their ghost role spawner is not working? I couldn't really tell you the original reason it was made this way though because I didn't add it.
Happy to replace it with CRASH if it seems more appropriate.

Reply:
I think that would be more appropriate yeah; ideally this should never be happening, and if it happens there should be a more permanent record.